simapp/app.go

```diff
@@ -182,7 +182,7 @@ func NewSimApp(
 		supply.NewAppModule(app.SupplyKeeper, app.AccountKeeper),
 		gov.NewAppModule(app.GovKeeper, app.SupplyKeeper),
 		mint.NewAppModule(app.MintKeeper),
-		distr.NewAppModule(app.DistrKeeper, app.SupplyKeeper),
+		distr.NewAppModule(app.DistrKeeper, app.SupplyKeeper, app.BankKeeper),
```

Comment:
Thanks for taking this on @RiccardoM! Note, the distribution should _not_ need access to the bank keeper. I haven't taken a look at the rest of the implementation, but you should be able to accomplish this feature w/o too many changes. In fact, I think you can just modify the black-listed accounts we provide to the bank keeper ([here](https://github.com/cosmos/cosmos-sdk/blob/master/simapp/app.go#L150)).

Correct me if I'm wrong here @fedekunze 

Remember, sending funds to the community pool is no different than sending funds to any other account. No special txs, commands, or REST handlers are needed.

That being said, we can provide a nicer client UX from the distribution module that simply does a `MsgSend` under the hood.

Also note, this needs to be done in Gaia too.

Reply:
@alexanderbez Thanks for the tip. I've added a new `BlacklistedAccAddrs` method so that the `ModuleAccAddrs` one (which is used also when creating a `distribution` keeper) is not modified. Should this be sufficient? 

Sorry but I didn't know about module accounts as I've never used them.